--- releases-CGAL-4.14.2/cmake/modules/CGAL_SetupFlags.cmake.orig	2023-09-10 21:58:21.976590136 +0200
+++ releases-CGAL-4.14.2/cmake/modules/CGAL_SetupFlags.cmake	2023-09-11 16:39:23.261285489 +0200
@@ -57,11 +57,6 @@ if ( NOT CMAKE_BUILD_TYPE )
   endif()
 endif()
 
-if ( NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Release" AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug"
-     AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo" AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "MinSizeRel" )
-  message( FATAL_ERROR "${CMAKE_BUILD_TYPE} is not a valid build type: only Release, Debug, RelWithDebInfo or MinSizeRel is allowed" )
-endif()
-
 message( STATUS "Build type: ${CMAKE_BUILD_TYPE}" )
 
 string( TOUPPER "${CMAKE_BUILD_TYPE}" CGAL_BUILD_TYPE_UPPER )
